pkgs/misc/uboot/default.nix

```diff
@@ -38,11 +45,13 @@ let
 
     version = if src == null then defaultVersion else version;
 
+    defconfig = if defconfig == null then
+      throw "uboot config not defined!"
+      else defconfig;
+
     src = if src == null then defaultSrc else src;
 
-    patches = [
-      ./0001-configs-rpi-allow-for-bigger-kernels.patch
-    ] ++ extraPatches;
+    patches = [] ++ extraPatches;
```

Comment:
The patches can and should be applied to all builds. It's built from the same source repo, so the patch applies equally just as well.